src/controller/CHIPDeviceController.cpp

```diff
@@ -664,6 +664,14 @@ CHIP_ERROR DeviceCommissioner::Shutdown()
 
     ChipLogDetail(Controller, "Shutting down the commissioner");
 
+    // Check to see if pairing in progress before shutting down
+    CommissioneeDeviceProxy * device = mDeviceBeingCommissioned;
+    if (device->IsSessionSetupInProgress())
```

Comment:
Shouldn't this check for `device` being null?

Reply:
Good catch, I'll add that check in there to avoid any edge cases.

Reply:
Resolved in e3674ebd7098a80cc9a74ffc5faf7a81774ac677